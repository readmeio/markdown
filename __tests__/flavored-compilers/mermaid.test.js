import { md } from '../../index';

describe('Compile Mermaid blocks', () => {
  it('compiles a mermaid block', () => {
    const value = `
graph LR
    A[Will it blend?] --> B[Yes]
`;

    const tree = {
      type: 'root',
      children: [
        {
          type: 'mermaid',
          value,
          data: {
            hName: 'mermaid',
            hProperties: {
              value,
            },
          },
        },
      ],
    };

    expect(md(tree)).toMatchSnapshot();
  });
});
