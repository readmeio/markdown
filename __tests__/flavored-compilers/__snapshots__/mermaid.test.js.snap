// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Compile Mermaid blocks compiles a mermaid block 1`] = `
"\`\`\`mermaid-render-svg
graph LR
    A[Will it blend?] -> B[Yes]
\`\`\`
"
`;
